onebox/start_onebox.sh

```diff
@@ -19,11 +19,22 @@ set -x -e
 cd "$(dirname "$0")/../"
 
 ulimit -c unlimited
-rm -rf logs/
-mkdir -p logs
+# rm -rf logs/
+# mkdir -p logs
+
+# The subdirectory can be set through the environment variable ONEBOX_WORK
+workspace=$ONEBOX_WORK
 
 IP=127.0.0.1
 
+# work unified file directory
+if [ ! -n "$workspace" ]; then
+  # default work directory
+  workspace="workspace"
+  rm -rf $workspace/logs/
+  mkdir -p $workspace/logs
```

Comment:
should mkdir for other `workspace` too

Reply:
ok

Reply:
review code ：[https://github.com/4paradigm/OpenMLDB/pull/1899/commits/760ff33219d39661801bbcd212dd7ba8830a8954](https://github.com/4paradigm/OpenMLDB/pull/1899/commits/760ff33219d39661801bbcd212dd7ba8830a8954)

Reply:
> should mkdir for other `workspace` too

review code ：[https://github.com/4paradigm/OpenMLDB/pull/1899/commits/760ff33219d39661801bbcd212dd7ba8830a8954](https://github.com/4paradigm/OpenMLDB/pull/1899/commits/760ff33219d39661801bbcd212dd7ba8830a8954)